### PR TITLE
Sturpin 2016 04 06 cert api

### PR DESCRIPTION
**Pull request by stenwt**

### CODE REVIEW

File: scripts/certificates/certificate-generate-and-request.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        parser.add_argument('--debug', action='store_true', default=None, help='Debug?')
+
+        self.args = parser.parse_args()
+
+    def parse_config(self):
+        """ parse config file for additional info needed to generate the cert """
+
+        configfile = open(self.args.configfile, 'r')
+        config = yaml.load(configfile)
+        self.config = config
```
Comment (mwoodson) [inline scripts/certificates/certificate-generate-and-request.py]: why not self.confg = yaml.load(configfile)

It would combine the lines?

**Comment by mmahut** [comment]
Looks good.

File: scripts/certificates/certificate-generate-and-request.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            import logging
+            logging.basicConfig()
+            logging.getLogger().setLevel(logging.DEBUG)
+            requests_log = logging.getLogger("requests.packages.urllib3")
+            requests_log.setLevel(logging.DEBUG)
+            requests_log.propagate = True
+
+        url = self.args.digicert_api_url + "enterprise/certificate/ssl"
+        response = requests.post(url, headers=api_headers, data=digicert_csr_json)
+        import pprint
```
Comment (mwoodson) [inline scripts/certificates/certificate-generate-and-request.py]: I think we should move imports like this to the top, with other imports

**Comment by mwoodson** [comment]
:+1:

**Pull request merged**